tests/js/harness.js

```diff
@@ -7,10 +7,15 @@ require("gpii-express");
 require("../../");
 
 
-fluid.defaults("gpii.pouch.tests.harness", {
+fluid.defaults("gpii.test.pouch.harness", {
     gradeNames: ["fluid.component"],
-    port:       6789,
-    baseUrl:    "http://localhost:6789/",
+    databases: {
```

Comment:
Please remove your own databases from this grade and put the remainder into a reusable base grade in lib/